packages/headless/src/controllers/smart-snippet-questions-list/headless-smart-snippet-questions-list.ts

```diff
@@ -33,12 +28,28 @@ export interface SmartSnippetQuestionsList extends Controller {
   /**
    * Expand the specified snippet suggestion.
    *
+   * @param identifier - The `questionAnswerId` of the smart snippet to expand.
```

Comment:
```suggestion
   * @param identifier - The unique identifier of the smart snippet to expand.
```

This suggestion would make sense if we're not tied to the `questionAnswerId` entity 🙏